### ISSUE TITLE
Account should not be passed into the methods to call the functions

### ISSUE THREAD

**Issue by mingyeow** [KyberNetwork/fpr-sdk.js#36]
there will be overlap between the web3 instance and the account being passed in. 

the calling address should be passed in for example, but probably not the full web3 account

**Comment by mingyeow** [comment]
@cyberliem @secmask  -> we only need the caller address when calling the contract functions right? 

Why do we need the account per se?

**Comment by secmask** [comment]
I see account pass in for using `account.address` only, so we can reduce to not pass full account.

**Comment by mingyeow** [comment]
yup, we do need to asset permissions to make sure that the address is both in the web3 instance, as well as authorized to execute as admins.
